packs/BP/scripts/network.ts

```diff
@@ -89,123 +93,152 @@ export class MachineNetwork extends DestroyableObject {
    * automatically sets each generator's storage to the amount it sent that was not received.
    * returns automatically if the object is not valid.
    */
-  private *send(): Generator<void, void, void> {
+  private *send(): Generator<undefined, void, unknown> {
     if (!this.isValid) return;
```

Comment:
Use a more specific type for `TNext` (`Generator<T, TReturn, TNext>`) here like `void` or `undefined` instead of `unknown`.